src/com/xilinx/rapidwright/design/tools/RelocationTools.java

```diff
@@ -0,0 +1,272 @@
+package com.xilinx.rapidwright.design.tools;
```

Comment:
With new source files, we should probably add the license header https://github.com/Xilinx/RapidWright/blob/master/doc/SOURCE_HEADER.TXT

Reply:
A setting in the IDE can probably do this automatically.